scIB/metrics/metrics.py

```diff
@@ -199,14 +207,13 @@ def metrics(
 
     # clustering
     if nmi_ or ari_:
-        print('Clustering...')
         res_max, nmi_max, nmi_all = opt_louvain(
             adata_int,
             label_key=label_key,
             cluster_key=cluster_key,
             function=nmi,
             plot=False,
-            verbose=verbose,
+            verbose=False,
```

Comment:
why remove verbosity passing here?

Reply:
The output wasn't very informative (it's only preprocessing), so I removed it

Reply:
Maybe it's sth for a future "debug" option for verbosity. Let's leave for now.